### PR TITLE
Update two functions in the kinematics.c 

### PR DESCRIPTION
**Pull request by DavidLin1577**
1.For further expansion, use switch(x) {...} instead of if(x){...}else if(y){...} in kinematics_create();
2.Replace for() with  rt_memcpy in functionkinematics_get_rpm().

### CODE REVIEW

File: kinematics/kinematics.c [hunk 1]
```diff
@@ -30,23 +30,22 @@ kinematics_t kinematics_create(enum base k_base, float length_x, float length_y,
...
+    case TWO_WD:
         new_kinematics->total_wheels = 2;
-    }
-    if(k_base == FOUR_WD)
-    {
-        new_kinematics->total_wheels = 4;
-    }
-    if(k_base == ACKERMANN)
-    {
+	    break;
```
Comment (sogwms) [inline kinematics/kinematics.c]: !!! "tab"

File: kinematics/kinematics.c [hunk 2]
```diff
@@ -111,39 +110,29 @@ void kinematics_get_rpm(struct kinematics kin, struct velocity target_vel, rt_in
...
-    else if(kin.k_base == FOUR_WD)
-    {
-        res_rpm[0] = cal_rpm.motor1;
-        res_rpm[1] = cal_rpm.motor2;
-        res_rpm[2] = cal_rpm.motor3;
-        res_rpm[3] = cal_rpm.motor4;
-    }
-    else if(kin.k_base == ACKERMANN)
-    {
+        rt_memcpy(rpm, &res_rpm[0], sizeof(res_rpm));
```
Comment (sogwms) [inline kinematics/kinematics.c:118]: Given that all cases have the same snippet, it can be further simplified

Comment (DavidLin1577) [reply kinematics/kinematics.c:118]: ok

File: kinematics/kinematics.c (hunk 1 above)
Comment (DavidLin1577) [reply kinematics/kinematics.c]: ok, this is not 'tab', pinch in when use 'swith(){case x: break}'

**Pull request closed without merging**